content/docs/atplatform/specification.md

```diff
@@ -271,158 +293,100 @@ The `cram` verb follows the `from` verb. As an owner of the atServer, you should
-Following regex represents the syntax of the `scan` verb:
-```r'^scan$|scan(:showhidden:(?<showhidden>true|false))?(:(?<forAtSign>@[^:@\s]+))?(:page:(?<page>\d+))?( (?<regex>\S+))?$'```
-
-**Example:**
-
-`scan`
-
-`scan:showhidden:true`
-
-`scan: <regex>`
-
-**Response:**
-
-The atServer should return the keys within the atServer if the scan verb executed successfully. The atServer will respond accordingly to whether the atSign is authenticated or not.
-```data:[<keys>]```
+`<digest>` Required: Yes. Description: Encrypted challenge
 
-### The `update` verb
+### The `pol` verb
 
-**Synopsis:**
+**Synopsis**:
 
-The `update` verb is used to insert key/value pairs into a Key Store. An update can only be run by the owner of an atServer on his/her own atServer.
+The `pol` verb is part of the `pkam` process to authenticate oneself while connecting to someone else's atServer. The term 'pol' means 'proof of life' as it provides a near realtime assurance that the requestor is who it claims to be.
 
-Following regex represents the syntax of the `update` verb:
+Following regex represents the syntax of the `pol` verb:
 
-`r'^update:(?:ttl:(?<ttl>\d+):)?(?:ttb:(?<ttb>\d+):)?(?:ttr:(?<ttr>(-?)\d+):)?(ccd:(?<ccd>true|false):)?((?:public:)|(@(?<for@sign>[^@:\s]-):))?(?<atKey>[^:@]((?!:{2})[^@])+)(?:@(?<@sign>[^@\s]-))? (?<value>.+$)'`
+`r'^pol$'`
 
 **Example:**
 
-`update:location@bob bob's location value`
-
-`update:ttl:600000:location@bob bob's location value but key expires in 10 minutes`
-
-`update:@alice:phone@bob bob's phone number shared to @alice`
+`pol`
 
-**Response:**
+**Response:**:
 
-The atServer should return the commit id from Commit Log if the update is successful.
+If the user gets the challenge right the prompt should change to the atSign of the user.
 
-`data:<CommitId>`
+`<@sign>@`
 
-If the user provides the invalid update command, then it should respond with the following error and close the connection to the server
+If the user gets the cram authentication wrong, then it should respond back with the following error and close the connection to the server.
 
-`error:AT0003-Invalid Syntax`
+`error:AT0401-Client authentication failed`
 
 **Description:**
 
-The `update` verb should be used to perform create/update operations on the atServer. The `update` verb requires the owner of the atServer to authenticate himself/herself to the atServer using `from` and `cram` verbs.
-
-If a key has been created for another atSign user, the atServer should honor "autoNotify" configuration parameter.
+The `pol` verb follows the `from` verb. 'pol' indicates another atServer that the user who is trying to connect is ready to authenticate himself. For example, if @bob is trying to connect to @alice, @bob would take the key and value from the proof response of the verb and create a public key and value which then can be looked up by @alice. After @alice looks up @bob's atServer, @alice's atServer should change the prompt to @bob.
```

Comment:
'user' is confusing here.

The info that needs to be clearly conveyed is:
from+cram : Client program connecting to the atServer
from+pkam: Client program connecting to the atServer
from+pol: atServer connecting to another atServer

Reply:
changes made here:

- https://github.com/atsign-foundation/docs.atsign.com/pull/247/commits/5ad166d0c1af51e1d17aba72db260528bea554d2
- https://github.com/atsign-foundation/docs.atsign.com/pull/247/commits/f2a534e8b0fd215e0332fedbf2197d308e681272